src/Expectation.php

```diff
@@ -327,6 +327,36 @@ public function toEndWith(string $expected): Expectation
         return $this;
     }
 
+    /**
+     * Asserts that $number matches value's Length.
+     */
+    public function toHaveLength(int $number): Expectation
+    {
+        if (is_string($this->value)) {
+            Assert::assertEquals($number, grapheme_strlen($this->value));
+
+            return $this;
+        }
+
+        if (is_iterable($this->value)) {
+            return $this->toHaveCount($number);
+        }
+
+        if (is_object($this->value)) {
+            if (method_exists($this->value, 'toArray')) {
+                $array = $this->value->toArray();
+            } else {
+                $array = (array) $this->value;
+            }
+
+            Assert::assertCount($number, $array);
```

Comment:
return $this->toHaveCount($number);

Reply:
Updated. thanks

Reply:
Actually, I prefer your solution. Can you revert?

Reply:
Done! Thanks